packages/integration-karma/test/rendering/element-orders/index.spec.js

```diff
@@ -0,0 +1,45 @@
+import { createElement } from 'lwc';
+import Container from 'x/container';
+import WithDynamic from 'x/withDynamic';
+import WithEach from 'x/withEach';
+
+describe('diffing algo', () => {
+    it('should render slot default elements in correct order', function () {
+        const elm = createElement('x-container', { is: Container });
+        document.body.appendChild(elm);
+
+        elm.triggerIssue();
```

Comment:
It would be meaning full to check the initial `textContent` before `triggerIssue` is invoked. 

I would also recommend renaming the `triggerIssue` to something more generic. Without the context of this PR, we don't have any information about the issue you are attempting to fix here.